prov/psm2/src/psmx2_domain.c

```diff
@@ -227,12 +227,67 @@ STATIC int psmx2_domain_control(fid_t fid, int command, void *arg)
 	return 0;
 }
 
+static int psmx2_domain_ops_get_val(struct fid *fid, int var, void *val)
+{
+        struct psmx2_fid_domain *domain;
+ 
+	if (!val)
+		return -FI_EINVAL;
+
+	domain = container_of(fid, struct psmx2_fid_domain,
+			      util_domain.domain_fid.fid);
+        if (domain->util_domain.domain_fid.fid.fclass != FI_CLASS_DOMAIN)
+                return -FI_EINVAL;
+ 
+        switch (var) {
+        case FI_PSM2_DISCONNECT:
+                *(uint32_t *)val = domain->params.disconnect;
+                break;
+	}
+
+	return 0;
+}
+
+static int psmx2_domain_ops_set_val(struct fid *fid, int var, void *val)
+{
+        struct psmx2_fid_domain *domain;
+ 
+	if (!val)
+		return -FI_EINVAL;
+
+	domain = container_of(fid, struct psmx2_fid_domain,
+			      util_domain.domain_fid.fid);
+        if (domain->util_domain.domain_fid.fid.fclass != FI_CLASS_DOMAIN)
+                return -FI_EINVAL;
+ 
+        switch (var) {
+        case FI_PSM2_DISCONNECT:
+                domain->params.disconnect = *(uint32_t *)val;
+                break;
+	}
+
+	return 0;
+}
```

Comment:
There are some spaces instead of tabs in the above code.  You can see it on github fairly clearly.

Reply:
Fixed.